core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1665,6 +1665,7 @@ class KafkaConfig private(doLog: Boolean, val props: java.util.Map[_, _], dynami
   def usesSelfManagedQuorum: Boolean = processRoles.nonEmpty
 
   val migrationEnabled: Boolean = getBoolean(KafkaConfig.MigrationEnabledProp)
+  def enableZkApiForwarding: Boolean = migrationEnabled && interBrokerProtocolVersion.isApiForwardingEnabled
```

Comment:
please don't add a reference to `KafkaConfig.interBrokerProtocolVersion` here. That is going away since we are moving away from statically configuring IBP (or indeed calling it IBP)